### DISCUSSION TITLE
47F1CA11-1AD8-B598-1D56-036205D845C6

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#122] [General]
# 47F1CA11-1AD8-B598-1D56-036205D845C6

TOI-1690 (better known as WD 1856+534) is a white dwarf about 80.737 light-years away in the constellation Draco. It is part of a triple star system called G 229-20. This stellar remnant has a Jupiter-sized planet that apparently had survived its star's red giant phase

https://celestia.mobi/addon?item=47F1CA11-1AD8-B598-1D56-036205D845C6

**Comment by exocritic5457** [comment]
This planet turns green when I zoom out far enough and I don't know if it's a "me" problem or not.
